.gitignore

```
@@ -35,4 +35,6 @@ build/
 
 
 ## others
-run.sh
\ No newline at end of file
+run.sh
```

This prevents that other file being readded later
```suggestion
**/.DS_Store
run.sh
```